politeiawww/api/v1/v1.go

```diff
@@ -55,7 +56,6 @@ const (
 	RouteStartVote                = "/proposals/startvote"
 	RouteActiveVote               = "/proposals/activevote" // XXX rename to ActiveVotes
 	RouteCastVotes                = "/proposals/castvotes"
-	RouteUserCommentsLikes        = "/user/proposals/{token:[A-z0-9]{64}}/commentslikes"
```

Comment:
Is this going to break the GUI?

Reply:
Just checked the GUI, and we still use that route. Hope this helps :)

Reply:
The route is still there.  I just moved it up so that it's grouped with the other `/user/proposals/` routes.